connect/settings.py

```diff
@@ -74,11 +74,13 @@ def MIDDLEWARE_CLASSES(self):
             'NAME': os.getenv('DB_NAME'),
             'USER': os.getenv('DB_USER'),
             'PASSWORD': os.getenv('DB_PASSWORD'),
-            'HOST': os.getenv('DB_HOST', '127.0.0.1'),
+            'HOST': os.getenv('DB_HOST', 'localhost'),
             'PORT': os.getenv('DB_PORT', '5432'),
         },
     }
 
+    print(DATABASES)
```

Comment:
Well I am trying to figure out what's wrong, it is not passing yet but I don't know why.